clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/client/OpenstackClientProvider.groovy

```diff
@@ -156,6 +209,24 @@ abstract class OpenstackClientProvider {
   }
 
   /**
+   * Handler for an Openstack4J request with error common handling.
+   * @param closure makes the needed Openstack4J request
+   * @return returns the result from the closure
+   */
+  def handleRequest(Closure closure) {
```

Comment:
Is this a second `handleRequest`? Do we plan on removing the other one in a follow up PR?